src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -188,15 +189,28 @@ const keriNotificationsChangeHandler = async (
   event: KeriNotification,
   dispatch: ReturnType<typeof useAppDispatch>
 ) => {
-  dispatch(
-    setQueueConnectionCredentialRequest({
-      id: event?.id,
-      type: ConnectionCredentialRequestType.CREDENTIAL_OFFER_RECEIVED,
-      logo: "", // TODO: must define Keri logo
-      label: "Credential Issuance Server", // TODO: must define it
-      source: ConnectionType.KERI,
-    })
-  );
+  if (!event) {
```

Comment:
When does this happen? `event` is not an optional parameter - can it be `null` from Signify?

Reply:
Ah, I will move this check to another place.